# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../udftools/gcc43.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- udftools-1.0.0b3/pktsetup/pktsetup.c.vanilla	2009-02-13 12:45:46.000000000 +0100
+++ udftools-1.0.0b3/pktsetup/pktsetup.c	2009-02-13 12:46:09.000000000 +0100
@@ -21,6 +21,7 @@
 #include <sys/ioctl.h>
 #include <unistd.h>
 #include <getopt.h>
+#include <limits.h>
 #include <bits/types.h>
 #include <sys/types.h>
 
--- udftools-1.0.0b3/cdrwtool/cdrwtool.c.vanilla	2009-02-13 12:45:01.000000000 +0100
+++ udftools-1.0.0b3/cdrwtool/cdrwtool.c	2009-02-13 12:45:33.000000000 +0100
@@ -24,6 +24,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <limits.h>
 #include <unistd.h>
 #include <getopt.h>
 #include <signal.h>
